Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -1203,6 +1203,26 @@ def test_is_demisto_version_build_ge(mocker):
     assert not is_demisto_version_ge('5.5.0', '50001')
 
 
+def is_demisto_version_ge_an_int(mocker):
+    """
+    Given:
+        A server with a build lower than 1000000
+    When:
+        is_demisto_version_ge is running
+    Then:
+        Make sure the return is false, and does not treat them as string".
+    """
+    mocker.patch.object(
+        demisto,
+        'demistoVersion',
+        return_value={
+            'version': '6.1.0',
+            'buildNumber': '999999'
+        }
+    )
+    assert not is_demisto_version_ge('6.1.0', build_number='1000000')
```

Comment:
Why not add to the previous UT an assertion like this:

assert not is_demisto_version_ge('6.0.0', '400000')

Reply:
I remove this function and I added to the previous function, this case ***is_demisto_version_ge('6.0.0', '6')***, That in strings comparison it's bigger than build_number '500000' but in integers comparison it's lower